clusterloader2/pkg/measurement/util/ssh.go

```diff
@@ -45,8 +48,47 @@ type SSHResult struct {
 	Code   int
 }
 
+func execLocal(cmd string) (SSHResult, error) {
+	result := SSHResult{Cmd: cmd}
+	result.User = "localhost"
+
+	c := exec.Command("sh", "-c", cmd)
+	var outBuf, errBuf bytes.Buffer
+	c.Stdout = &outBuf
+	c.Stdout = &errBuf
+
+	if err := c.Start(); err != nil {
+		return result, fmt.Errorf("command start error %s: '%v'", cmd, err)
+	}
+	code := 0
+	if err := c.Wait(); err != nil {
+		if exiterr, ok := err.(*exec.ExitError); ok {
+			// The program has exited with an exit code != 0
+
+			// This works on both Unix and Windows. Although package
+			// syscall is generally platform dependent, WaitStatus is
+			// defined for both Unix and Windows and in both cases has
+			// an ExitStatus() method with the same signature.
+			if status, ok := exiterr.Sys().(syscall.WaitStatus); ok {
+				code = status.ExitStatus()
+			}
+		} else {
```

Comment:
Maybe we can invert the `if` to reduce the nesting?
```
v, ok := ...
if !ok {
    return err
}
do_stuff_with_v()
````

Reply:
Done. @kyzysied